habitatstreams/src/22_progress_evaluation_3260.Rmd

```diff
@@ -0,0 +1,522 @@
++ grts_ranking: grts-rangnummer van grts-hok waarbinnen referentiepunt gelegen is;
++ grts_ranking_draw: grts-rangnummer op basis waarvan de steekproeftrekking is gebeurd. In de meeste gevallen is dit gelijk aan de grts_ranking, maar als een referentiepunt verplaatst wordt omdat het originele referentiepunt niet geschikt is, behouden we de grts-ranking van het originele referentiepunt voor grts_ranking_draw ook als het nieuwe punt in een ander grts-hok gelegen is (met een andere grts-ranking);
++ legacy_site: gaat het om een al langer bestaand meetpunt? 
++ x- en y-coördinaten van het referentiepunt;
++ db_ref: een andere referentie voor een steekproefeenheid die in een databank gebruikt wordt.
+
+```{r}
+mhq_streams_refpoints <- sample_assessments_3260_update %>%
+  mutate(legacy_site = FALSE,
+         db_ref = field_code) %>%
+  select(scheme, sampling_unit_code, segment_id, point_code, type_target, sac_streams, grts_ranking, grts_ranking_draw, legacy_site, x, y, db_ref) %>%
+  arrange(grts_ranking_draw, point_code)
+```
+
+Als voorbeeld tonen we de eerste zes rijen van de tabel.
+
+```{r}
+mhq_streams_refpoints %>%
+  head() %>%
+  kable() %>%
+  kable_styling()
+```
+
+
+```{r}
+write_vc(mhq_streams_refpoints, file = "mhq_streams_refpoints", root = "../output", sorting = c("sampling_unit_code", "point_code"), strict = FALSE)
+```
+
+### mhq_streams_refpoints_validity
+
+De tabel 'mhq_streams_refpoints_validity' geeft aan welke steekproefeenheden voldoen aan de voorwaarden om opgenomen te worden in het biotisch meetnet habitatkwaliteit voor 3260.
+
+De tabel bestaat uit volgende variabelen:
+
++ scheme
++ sampling_unit_code
++ point_code
++ date: datum waarop de beoordeling is gebeurd
++ is_valid_type: komt het geobserveerde habitattype overeen met het doelhabitat?
++ is_valid_refpoint: is er een LSVI-meting mogelijke in het segment/referentiepunt?
++ is_valid: is het een geldig meetpunt? 
+
+'is_valid' is TRUE als zowel 'is_valid_type' als 'is_valid_refpoint' TRUE zijn. 
+
+
+```{r}
+mhq_streams_refpoints_validity <- sample_assessments_3260_update %>%
+  arrange(grts_ranking_draw, point_code) %>%
+  mutate(is_valid_type = is_type_target,
+         is_valid_refpoint = segment_measurable & is.na(inaccessible),
+         is_valid = is_valid_type & is_valid_refpoint) %>%
+  select(scheme, sampling_unit_code, point_code, date = assessment_date, is_valid, is_valid_type, is_valid_refpoint) 
+```
+
+Als voorbeeld tonen we de eerste zes rijen van de tabel.
+
+```{r}
+mhq_streams_refpoints_validity %>%
+  head() %>%
+  kable() %>%
+  kable_styling()
+
+```
+
+```{r}
+write_vc(mhq_streams_refpoints_validity, file = "mhq_streams_refpoints_validity", root = "../output", sorting = c("sampling_unit_code", "point_code"))
+```
+
+### mhq_streams_assessments
+
+De tabel 'mhq_streams_assessments' geeft een overzicht van de beoordelingen (assessments) van de steekrpoefeenheden.
+
+De tabel bevat volgende variabelen:
+
++ segment_id
++ point_code
```

Comment:
'point_code': rather define this table by using unique values of 'sampling_unit_code': it is sampling units which are assessed here (contrary to terrestrial situation, where the actual points are assessed as being the target type or not). So I suggest to replace 'point_code' by 'sampling_unit_code'.